src/runmode-dpdk.c

```diff
@@ -1596,8 +1596,7 @@ static void *ParseDpdkConfigAndConfigureDevice(const char *iface)
     if (iconf->workers_sync == NULL) {
```

Comment:
interesting, I didn't know about pthread_barrier. The commit needs a lot more explanation of what it does.

Reply:
ok, btw FYI it is not available for MacOS but I believe it fits the DPDK scenario.